src/test/mochitest/examples/doc-babel.html

```diff
@@ -29,6 +29,8 @@
     <button onclick="functions()">Run functions</button>
     <script src="babel/fixtures/imported-bindings/output.js"></script>
     <button onclick="importedBindings()">Run importedBindings</button>
+    <script src="babel/fixtures/line-start-bindings-es6/output.js"></script>
+    <button onclick="lineStartBindingsEs6()">Run lineStartBindingsEs6</button>
```

Comment:
It would be nice to add documentation here maybe so that it is clear what each case is for..

| example | description  |
| -- | -- |
| lineStartBindingsEs6 | tests es6 bindings on many lines to see if ... |
| ... | .. |

Presumably this doc would be with a `<table><tr><td>` markup :)

Reply:
Yeah sounds like a good idea.

Reply:
Should I just add that to the tracking issue?

Reply:
already did.